plugins/catalog-react/src/components/EntityDisplayName/EntityDisplayName.tsx

```diff
@@ -77,7 +77,16 @@ export const EntityDisplayName = (
   );
 
   // The innermost "body" content
-  let content = <>{primaryTitle}</>;
+  let content = (
+    <div
+      style={{
+        overflow: 'hidden',
+        wordBreak: 'break-word',
+      }}
```

Comment:
This should go into the `makeStyles` above, under a new key like `content` or so.

Reply:
@freben moved these styles under *makeStyles*